### PR TITLE
Fixed static height of Logs & terminal

### PR DESCRIPTION
**Pull request by milindk8**
# Description

Terminal and log div is not taking dynamic height & not acquiring full height of view

Fixes - https://github.com/devtron-labs/devtron/issues/1668

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
**Describe the bug**

Terminal and log div is not taking dynamic height & not acquiring full height of view 

**To Reproduce**

1. User goes to devtron apps or helm apps 
2. User selects pods & select terminal or logs
3. Terminal and logs is not taking full height

**Expected behavior**

Terminal and logs should take full available height

**Current behavior** 

Terminal and logs is not taking full height

**Screenshots**

# Checklist:

* [x] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [x] I have performed a self-review of my own code
* [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/components/app/details/appDetails/AppDetails.tsx [hunk 1]
```diff
@@ -102,8 +102,7 @@ export default function AppDetail() {
         setEnvironmentId(Number(params.envId));
     }, [params.envId]);
     return (
-        <div style={{ overflowY: 'auto', height: '100%' }}>
-            <div style={{ display: 'flex', flexDirection: 'column', flex: 1 }}>
+        <div style={{ display: 'flex', flexDirection: 'column', flex: 1 }}>
```
Comment (vivek-devtron) [inline src/components/app/details/appDetails/AppDetails.tsx:105]: Please use className

File: src/components/v2/appDetails/k8Resource/nodeDetail/NodeDetailTabs/LogViewer.component.tsx [hunk 2]
```diff
@@ -4,6 +4,7 @@ import { FitAddon } from 'xterm-addon-fit';
 import CopyToast, { handleSelectionChange } from './CopyToast';
```
Comment (vivek-devtron) [inline src/components/v2/appDetails/k8Resource/nodeDetail/NodeDetailTabs/LogViewer.component.tsx:4]: Please remove CopyToast if we are not using it

Comment (milindk8) [reply src/components/v2/appDetails/k8Resource/nodeDetail/NodeDetailTabs/LogViewer.component.tsx:4]: We are using it to show popup

File: src/components/app/details/appDetails/AppDetails.tsx (hunk 1 above)
Comment (milindk8) [reply src/components/app/details/appDetails/AppDetails.tsx]: Done

File: src/components/app/details/appDetails/AppDetails.tsx [hunk 3]
```diff
@@ -102,8 +102,7 @@ export default function AppDetail() {
         setEnvironmentId(Number(params.envId));
     }, [params.envId]);
     return (
-        <div style={{ overflowY: 'auto', height: '100%' }}>
-            <div style={{ display: 'flex', flexDirection: 'column', flex: 1 }}>
+        <div className="app-details-page-wrapper">
                 {/* <div className="flex left w-100 p-16">
```
Comment (vivek-devtron) [inline src/components/app/details/appDetails/AppDetails.tsx:106]: Remove commented code

Comment (milindk8) [reply src/components/app/details/appDetails/AppDetails.tsx]: done

**Review by vivek-devtron — APPROVED** [review]

**Pull request merged**